addons/binding/pom.xml

```diff
@@ -52,6 +52,8 @@
     <module>org.openhab.binding.mihome</module>
     <module>org.openhab.binding.milight</module>
     <module>org.openhab.binding.minecraft</module>
+    <module>org.openhab.binding.mysensors</module>
+    <module>org.openhab.binding.mysensors.test</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

Reply:
@tobof this point seems still to be open

Reply:
@martinvw, you mean [this feature.xml](https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml) ?

Reply:
Correct